### DISCUSSION TITLE
QValue categories

### DISCUSSION THREAD

**Discussion by tglatter** [vdemichev/DiaNN#395] [General]
Dear Vadim,
I have a question on how the different QValues relate to each other (see attached screen shot). I have a peptide I am interested in, with a QValue ranging from 0.000149 and 7.5E-5, Global QValue is 0.1, PG QValues around 0.04, Protein QV, Global PGQV and GGQV all 1. I am not sure how one should weight these different ranges across the QValue categories. Could you give a general comment what the basis for these differences are. 
Thanks (also in general for the great tool)
Best Timo
![image](https://user-images.githubusercontent.com/106140704/170220793-cc30a994-8674-40b4-b3fc-51dbf1927636.png)

**Comment by vdemichev** [comment]
Hi Timo, 

Please see https://www.nature.com/articles/nmeth.4398 on interpretation of different q-value scores. Please also see DIA-NN docs on recommendations how to filter the output. For most applications the default filtering used to generate the output matrices will be the best.

Best,
Vadim
